### PR TITLE
Hashtable implementation

### PR DESCRIPTION
**Pull request by nikitavbv**

### CODE REVIEW

File: src/main/java/com/labs/introtoprogramming/lab3/datastructures/MyListInterface.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+   */
+  public void set(T element, int index);
+
+  /**
+   * Find element in the list and remove it.
+   *
+   * @param element element to be removed
+   * @throws java.util.NoSuchElementException if element is not found
+   */
+  public void remove(T element);
```
Comment (Brookoko) [inline src/main/java/com/labs/introtoprogramming/lab3/datastructures/MyListInterface.java]: Duplicated method. Maybe it is find.

Comment (Brookoko) [inline src/main/java/com/labs/introtoprogramming/lab3/datastructures/MyListInterface.java]: Duplicated method. Maybe it is find.

Comment (nikitavbv) [reply src/main/java/com/labs/introtoprogramming/lab3/datastructures/MyListInterface.java]: In fact it is not :) It is convenient to have two `remove` methods, one of each removes element by index, while other just finds and removes element.

Comment (Brookoko) [reply src/main/java/com/labs/introtoprogramming/lab3/datastructures/MyListInterface.java]: Wow, okay. Now I know it.

Comment (nikitavbv) [reply src/main/java/com/labs/introtoprogramming/lab3/datastructures/MyListInterface.java]: I will change method names anyway...

**Pull request merged**